image/api_selector.go

```diff
@@ -12,6 +12,7 @@ import (
 
 	"github.com/cenk/backoff"
 	"github.com/pkg/errors"
+	workerErrors "github.com/travis-ci/worker/errors"
```

Comment:
Convention is to import as smashedlowercase, e.g. `workererrors`.  I don't recall if there's a lint check for this, though, and I'm assuming not...